src/main/java/org/ibp/api/brapi/v1/program/Program.java

```diff
@@ -2,21 +2,41 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonView;
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
 import org.apache.commons.lang3.builder.ToStringStyle;
+import org.generationcp.middleware.service.api.BrapiView;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({"programDbId", "name", "abbreviation", "objective", "leadPerson"})
+@JsonPropertyOrder({"programDbId", "name", "programName", "abbreviation", "objective", "leadPerson"})
 public class Program {
 
 	private String programDbId;
-	private String name;
 	private String abbreviation;
 	private String objective;
+
+	@JsonView({BrapiView.BrapiV1_2.class, BrapiView.BrapiV1_3.class})
+	private String name;
+	@JsonView({BrapiView.BrapiV1_2.class, BrapiView.BrapiV1_3.class})
 	private String leadPerson;
 
+	@JsonView(BrapiView.BrapiV2.class)
+	private String programName;
+	@JsonView(BrapiView.BrapiV2.class)
+	private String leadPersonDbId;
+	@JsonView(BrapiView.BrapiV2.class)
+	private String leadPersonName;
+	@JsonView(BrapiView.BrapiV2.class)
+	private String documentationURL;
+	@JsonView(BrapiView.BrapiV2.class)
+	private String externalReferences;
+	@JsonView(BrapiView.BrapiV2.class)
+	private String commonCropName;
+	@JsonView(BrapiView.BrapiV2.class)
+	private ProgramAdditionalInfo additionalInfo;
```

Comment:
I don't think you need ProgramAdditionalInfo class. can be just Map<String, String>